rust_src/src/frames.rs

```diff
@@ -103,6 +104,13 @@ impl LispFrameRef {
             0
         }
     }
+
```

Comment:
`LispObject` is already a kind of `Option`. Wrapping it in an `Option` does not buy us match currently. I'd probably write this:

```
map assq(prop, self.param_alist) {
    Qnil => Qnil,
    cons => cdr(cons),
}
```